<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Stefan Duffner</author>
 <class>ExportVVVVDlg</class>
 <widget class="QDialog" name="ExportVVVVDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vvvv Automata Export</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextBrowser" name="tb_output">
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_reset">
     <property name="text">
      <string>Add &quot;Reset&quot; transition for every state</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Reset event:</string>
       </property>
       <property name="buddy">
        <cstring>le_reset_event</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_reset_event">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Reset action:</string>
       </property>
       <property name="buddy">
        <cstring>le_reset_action</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_reset_action">
       <property name="text">
        <string>DoReset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pb_update">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>138</width>
         <height>29</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_close</sender>
   <signal>clicked()</signal>
   <receiver>ExportVVVVDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_reset</sender>
   <signal>clicked()</signal>
   <receiver>ExportVVVVDlg</receiver>
   <slot>resetClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_reset_event</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ExportVVVVDlg</receiver>
   <slot>resetEventChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_reset_action</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ExportVVVVDlg</receiver>
   <slot>resetActionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>resetClicked()</slot>
  <slot>updateClicked()</slot>
  <slot>resetEventChanged()</slot>
  <slot>resetActionChanged()</slot>
 </slots>
</ui>
